tasks/publish.yaml

```diff
@@ -5,60 +5,36 @@ variables:
   - name: FLAVOR
     default: upstream
 
+  - name: TYPE
+
   - name: VERSION
     description: "The version of the packages to build"
     # x-release-please-start-version
     default: "0.15.0"
     # x-release-please-end
 
 tasks:
-  - name: packages
+  - name: standard-package
     description: "Build and publish the UDS packages"
     actions:
-      - description: "Create the UDS Core Standard Zarf Package"
-        cmd: |
-          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${FLAVOR}
-          if [ "${FLAVOR}" != "registry1" ]; then
-            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${FLAVOR}
-          fi
-
-      - description: "Create the UDS Core Istio Only Zarf Package"
-        cmd: |
-          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml istio-package --no-progress --set FLAVOR=${FLAVOR}
-          if [ "${FLAVOR}" != "registry1" ]; then
-            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml istio-package --no-progress --set FLAVOR=${FLAVOR}
-          fi
-
       - description: "Publish the packages"
         cmd: |
-          uds zarf package publish build/zarf-package-core-amd64-${VERSION}.tar.zst ${TARGET_REPO}
-
-          # dont publish arm64 for registry1 since IB images are only amd64
+         ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${FLAVOR}
           if [ "${FLAVOR}" != "registry1" ]; then
-            uds zarf package publish build/zarf-package-core-arm64-${VERSION}.tar.zst ${TARGET_REPO}
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${FLAVOR}
           fi
```

Comment:
I think we dropped the wrong section here - looking at [the task](https://github.com/defenseunicorns/uds-core/blob/update-publish-ci/tasks/publish.yaml#L22-L25) (because the diffs are overwhelming) this isn't doing a publish anymore, just a create. Should drop the create and only do the publish I believe?

Reply:
ohhh, yep that's right. i only tested the publish bundle locally so i didn't catch that.